### PR TITLE
Update news.html

### PR DESCRIPTION
**Pull request by david4096**
Updates news

### CODE REVIEW

File: news.html [hunk 1]
```diff
@@ -1,4 +1,15 @@
...
+  <div class="news-entry py-3">
+    <div>
+      <h5 class="mb-0">
+          <a target="_blank" rel="noopener noreferrer"
+             href="https://github.com/dockstore/dockstore/releases/tag/1.16.0">
+              1.16 release</a>
+      </h5>
+        <span>Dockstore's latest version improves DOI generation and adds AI topic sentences. See more at GitHub!</span>
+    </div>
+    <span class="date-display ml-3">Feb 8, 2024</span>
```
Comment (coverbeck) [inline news.html:11]: Wrong date

Comment (david4096) [reply news.html]: Set to a guess on the date

File: news.html [hunk 2]
```diff
@@ -1,4 +1,15 @@
 <div class="news-and-updates-box h-100">
+  <div class="news-entry py-3">
+    <div>
+      <h5 class="mb-0">
+          <a target="_blank" rel="noopener noreferrer"
+             href="https://github.com/dockstore/dockstore/releases/tag/1.16.0">
+              1.16 release</a>
```
Comment (svonworl) [inline news.html:7]: Nit: would recommend using "title case" in header text.

**Review by svonworl — APPROVED** [review]

**Review by kathy-t — APPROVED** [review]

**Comment by david4096** [comment]
Closing in favor of https://github.com/dockstore/extra-content/pull/57#pullrequestreview-2419199786

**Pull request closed without merging**